### PR TITLE
Fix crashes introduced by d15808.

### LINKED ISSUE #60: Crash when clicking on opening
The distro I'm using is Arch Linux
Reverting d15808912fd80df4f57f4f72256a1083fb05ec9a fixes it for me.
Log is
```
Tmds.DBus.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name com.canonical.AppMenu.Registrar was not provided by any .service files
   at Tmds.DBus.DBusConnection.CallMethodAsync(Message msg, Boolean checkConnected, Boolean checkReplyType)
   at Tmds.DBus.Connection.CallMethodAsync(Message message)
   at Tmds.DBus.CodeGen.DBusObjectProxy.SendMethodReturnReaderAsync(String iface, String member, Nullable`1 inSignature, MessageWriter writer)
   at Avalonia.FreeDesktop.DBusMenuExporter.DBusMenuExporterImpl.Init() in /_/src/Avalonia.FreeDesktop/DBusMenuExporter.cs:line 59
Tmds.DBus.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name com.canonical.AppMenu.Registrar was not provided by any .service files
   at Tmds.DBus.DBusConnection.CallMethodAsync(Message msg, Boolean checkConnected, Boolean checkReplyType)
   at Tmds.DBus.Connection.CallMethodAsync(Message message)
   at Tmds.DBus.CodeGen.DBusObjectProxy.SendMethodReturnReaderAsync(String iface, String member, Nullable`1 inSignature, MessageWriter writer)
   at Avalonia.FreeDesktop.DBusMenuExporter.DBusMenuExporterImpl.Init() in /_/src/Avalonia.FreeDesktop/DBusMenuExporter.cs:line 59
Tmds.DBus.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name com.canonical.AppMenu.Registrar was not provided by any .service files
   at Tmds.DBus.DBusConnection.CallMethodAsync(Message msg, Boolean checkConnected, Boolean checkReplyType)
   at Tmds.DBus.Connection.CallMethodAsync(Message message)
   at Tmds.DBus.CodeGen.DBusObjectProxy.SendMethodReturnReaderAsync(String iface, String member, Nullable`1 inSignature, MessageWriter writer)
   at Avalonia.FreeDesktop.DBusMenuExporter.DBusMenuExporterImpl.Init() in /_/src/Avalonia.FreeDesktop/DBusMenuExporter.cs:line 59
Sorry, we crashed
System.ArgumentNullException: String reference not set to an instance of a String. (Parameter 's')
   at System.Text.Encoding.GetBytes(String s)
   at Avalonia.X11.X11Window.SetTitle(String title) in /_/src/Avalonia.X11/X11Window.cs:line 1002
   at Avalonia.Controls.Window.<>c.<.cctor>b__26_0(Window s, AvaloniaPropertyChangedEventArgs e) in /_/src/Avalonia.Controls/Window.cs:line 199
   at Avalonia.AvaloniaObjectExtensions.<>c__DisplayClass23_0`1.<AddClassHandler>b__0(AvaloniaPropertyChangedEventArgs e) in /_/src/Avalonia.Base/AvaloniaObjectExtensions.cs:line 585
   at System.Reactive.Subjects.Subject`1.OnNext(T value) in /_/Rx.NET/Source/src/System.Reactive/Subjects/Subject.cs:line 147
   at Avalonia.AvaloniaObject.RaisePropertyChanged[T](AvaloniaPropertyChangedEventArgs`1 change) in /_/src/Avalonia.Base/AvaloniaObject.cs:line 720
   at Avalonia.AvaloniaObject.Avalonia.PropertyStore.IValueSink.ValueChanged[T](AvaloniaPropertyChangedEventArgs`1 change) in /_/src/Avalonia.Base/AvaloniaObject.cs:line 493
   at Avalonia.ValueStore.SetValue[T](StyledPropertyBase`1 property, T value, BindingPriority priority) in /_/src/Avalonia.Base/ValueStore.cs:line 103
   at Avalonia.AvaloniaObject.SetValue[T](StyledPropertyBase`1 property, T value, BindingPriority priority) in /_/src/Avalonia.Base/AvaloniaObject.cs:line 352
   at Avalonia.Controls.Window.set_Title(String value) in /_/src/Avalonia.Controls/Window.cs:line 276
   at Avalonia.Dialogs.ManagedFileDialogExtensions.ManagedSystemDialogImpl`1.Show(SystemDialog d, Window parent, ManagedFileDialogOptions options) in /_/src/Avalonia.Dialogs/ManagedFileDialogExtensions.cs:line 17
   at Avalonia.Dialogs.ManagedFileDialogExtensions.ManagedSystemDialogImpl`1.ShowFileDialogAsync(FileDialog dialog, Window parent) in /_/src/Avalonia.Dialogs/ManagedFileDialogExtensions.cs:line 42
   at ICSharpCode.ILSpy.MainWindow.OpenCommandExecuted(Object sender, ExecutedRoutedEventArgs e) in /mnt/void/home/archdata/cose/AvaloniaILSpy/ILSpy.Core/MainWindow.xaml.cs:line 1105
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_0(Object state)
   at Avalonia.Threading.AvaloniaSynchronizationContext.<>c__DisplayClass5_0.<Post>b__0() in /_/src/Avalonia.Base/Threading/AvaloniaSynchronizationContext.cs:line 33
   at Avalonia.Threading.JobRunner.RunJobs(Nullable`1 priority) in /_/src/Avalonia.Base/Threading/JobRunner.cs:line 37
   at Avalonia.X11.X11PlatformThreading.HandleX11(CancellationToken cancellationToken)
   at Avalonia.X11.X11PlatformThreading.RunLoop(CancellationToken cancellationToken) in /_/src/Avalonia.X11/X11PlatformThreading.cs:line 245
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken) in /_/src/Avalonia.Base/Threading/Dispatcher.cs:line 62
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 108
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime[T](T builder, String[] args, ShutdownMode shutdownMode) in /_/src/Avalonia.Controls/ApplicationLifetimes/ClassicDesktopStyleApplicationLifetime.cs:line 134
   at ICSharpCode.ILSpy.Program.Main(String[] args) in /mnt/void/home/archdata/cose/AvaloniaILSpy/ILSpy/Program.cs:line 22
```

### PR DESCRIPTION
**Pull request by AaronDewes**
Fixes #60.

This adds a simple title to every dialog to prevent crashes.
I'm not sure about your coding standards, yoem to be using tabs & spaces mixed :man_shrugging:?

### CODE REVIEW

**Comment by christophwille** [comment]
What crashes?

**Comment by AaronDewes** [comment]
Sorry, I forgot to link the issue, this fixes #60. I edited the original description now

File: ILSpy.Core/Commands/OpenFromNuGetCommand.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@ sealed class OpenFromNuGetCommand : SimpleCommand
 		public override async void Execute(object parameter)
 		{
             OpenFileDialog dlg = new OpenFileDialog();
+			dlg.Title = "Open file";
```
Comment (MarijnS95) [inline ILSpy.Core/Commands/OpenFromNuGetCommand.cs:33]: Looking at the coding style, this file uses a mix of spaces and tabs, but the surrounding context of this line is indented with spaces and so should this. Might be worth enabling "render whitespace" (or equivalent) in your editor to see what is going on and adjust accordingly.

File: ILSpy.Core/Commands/SaveCodeContextMenuEntry.cs [hunk 2]
```diff
@@ -95,6 +95,7 @@ public static async Task Execute(IReadOnlyList<SharpTreeNode> selectedNodes)
         static async Task<string> SelectSolutionFile()
         {
             SaveFileDialog dlg = new SaveFileDialog();
+			dlg.Title = "Save file";
```
Comment (MarijnS95) [inline ILSpy.Core/Commands/SaveCodeContextMenuEntry.cs:98]: This one uses spaces, too.

File: ILSpy.Core/TextView/DecompilerTextView.xaml.cs [hunk 3]
```diff
@@ -686,6 +686,7 @@ public async void SaveToDisk(Language language, IEnumerable<ILSpyTreeNode> treeN
                 return;
 
             SaveFileDialog dlg = new SaveFileDialog();
+			dlg.Title = "Save file";
```
Comment (MarijnS95) [inline ILSpy.Core/TextView/DecompilerTextView.xaml.cs:689]: And more spaces.

File: ILSpy.Core/TreeNodes/ResourceNodes/ResourceTreeNode.cs [hunk 4]
```diff
@@ -108,6 +108,7 @@ public override async Task<bool> Save(DecompilerTextView textView)
 			if (s == null)
 				return false;
             SaveFileDialog dlg = new SaveFileDialog();
+			dlg.Title = "Save file";
```
Comment (MarijnS95) [inline ILSpy.Core/TreeNodes/ResourceNodes/ResourceTreeNode.cs:111]: Spaces.

File: ILSpy.Core/TreeNodes/ResourceNodes/ResourcesFileTreeNode.cs [hunk 5]
```diff
@@ -117,6 +117,7 @@ public override async Task<bool> Save(DecompilerTextView textView)
 			Stream s = Resource.TryOpenStream();
 			if (s == null) return false;
             SaveFileDialog dlg = new SaveFileDialog();
+			dlg.Title = "Save file";
```
Comment (MarijnS95) [inline ILSpy.Core/TreeNodes/ResourceNodes/ResourcesFileTreeNode.cs:120]: Spaces.

**Review by MarijnS95 — APPROVED** [review]
Mixed indentation is always a pain to deal with, but at least this PR fixes the crash when simply trying to open a file, thanks! :+1: 

(I wonder why the CI couldn't do something as simple as running and enforcing a certain formatting style)

**Pull request merged**